ore/public/javascripts/apiRequests.js

```diff
@@ -3,31 +3,23 @@
 let csrf = null;
 let isLoggedIn = false;
 
+//=====> SETUP
+
+jQuery.ajaxSettings.traditional = true;
+
 //=====> HELPER FUNCTIONS
 
-function apiV2Request(url, method, data, isRetry) {
+function apiV2Request(url, method = "GET", data = {}, isRetry = true) {
```

Comment:
From what I remember when I wrote this function, isRetry should only be true when it has already tried the request once, failed, and gotten a new token. As it is now it breaks loading if the user isn't logged in.